3rdparty/jvm/BUILD

```diff
@@ -12,6 +12,6 @@ jar_library(name = 'protobuf-runtime',
 
 jar_library(name = 'scalapb-runtime-grpc',
   jars = [
-    jar('com.thesamet.scalapb', 'scalapb-runtime-grpc_2.12', '0.9.4')
+    jar('com.thesamet.scalapb', 'scalapb-runtime-grpc_2.12', '0.10.1')
```

Comment:
Can we move all the way to latest (0.10.7)?

Reply:
Oops! sorry. I've checked the website https://scalapb.github.io/scalapbc.html and saw `0.10.1` there. Should've checked maven repo instead. Now it is fixed. I've also tried to resolve Travis issues and explicitly set the JVM platform to 11 in both Pants TOML and Travis config.